doc/api/util.md

```diff
@@ -1380,15 +1380,15 @@ added:
   - v18.3.0
   - v16.17.0
 changes:
-  - version: v18.11.0
```

Comment:
This change does not seem correct. We put the most recent change first.

Reply:
The original issue is the result of the sorting done here https://github.com/nodejs/node/blob/f6052c68c1f9a4400a723e9c0b79da14197ab754/tools/doc/html.mjs#L348

The versions in the `meta.changes` array are

```js
['v18.11.0', ['v18.7.0', 'v16.17.0'], 'v18.3.0']
```

and [`versionSort()`](https://github.com/nodejs/node/blob/f6052c68c1f9a4400a723e9c0b79da14197ab754/tools/doc/html.mjs#L388-L396) uses the min version (`'v16.17.0'`), so the array is sorted as follows:

```js
['v18.11.0', 'v18.3.0', ['v18.7.0', 'v16.17.0']]
```